packit/config/aliases.py

```diff
@@ -0,0 +1,129 @@
+# MIT License
+#
+# Copyright (c) 2018-2019 Red Hat, Inc.
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+from typing import Dict, List, Set
+
+from packit.exceptions import PackitException
+
+ALIASES: Dict[str, List[str]] = {
+    "fedora-development": ["fedora-rawhide", "fedora-32"],
+    "fedora-stable": ["fedora-30", "fedora-31"],
+    "fedora-all": ["fedora-rawhide", "fedora-32", "fedora-30", "fedora-31"],
+}
+ARCHITECTURE_LIST: List[str] = [
+    "aarch64",
+    "armhfp",
+    "i386",
+    "ppc64le",
+    "s390x",
+    "x86_64",
+]
+DEFAULT_VERSION = "fedora-stable"
+
+
+def get_versions(*name: str, default=DEFAULT_VERSION) -> Set[str]:
+    """
+    Expand the aliases to the name(s).
+
+    :param name: name(s) of the system and version (e.g. "fedora-30"/"fedora-stable")
+    :param default: used if no positional argument was given
+    :return: set of string containing system name and version
+    """
+    names = list(name) or [default]
+    versions: Set[str] = set()
+    for one_name in names:
+        versions.update(ALIASES.get(one_name, [one_name]))
+    return versions
+
+
+def get_build_targets(*name: str, default=DEFAULT_VERSION) -> Set[str]:
+    """
+    Expand the aliases to the name(s) and transfer to the build targets.
+
```

Comment:
the code in this function is a little cryptic, so a bit of docs on what's happening would be appreciated